extensions/layout/src/org/sbml/jsbml/ext/layout/Layout.java

```diff
@@ -502,11 +502,11 @@ public boolean equals(Object object) {
    */
   @SuppressWarnings("unchecked")
   private <T> List<T> findGlyphs(ListOf<? extends T> listOfGlyphs, String id) {
-    // TODO - use the Model findUniqueNamedSBase method if model is not null ?
+    // TODO 2014/05: use the Model findUniqueNamedSBase method if model is not null ?
     if (isSetListOfReactionGlyphs() && (listOfGlyphs != null) && (!listOfGlyphs.isEmpty())) {
```

Comment:
You need to remove the `isSetListOfReactionGlyphs()` test as well. You can remove the TODO as well. 

The javadoc should be written as well to explain that what we return are glyphs that reference the given id. We are not looking for a glyph id here which might be confusing.